index.src.html

```diff
@@ -1704,6 +1717,9 @@
 
 dictionary SvcOutputMetadata {
   unsigned long temporalLayerId;
+  unsigned long spatialLayerId;
+  unsigned long long frameId;
```

Comment:
`frameId` and `dependencies` are never set

Reply:
@Djuffin @tonyherre `dependencies` would be set to the sequence of `frameId` values that the `encodedChunk` depends on. But the bigger question is the behavior of `frameId`, which is discussed here: https://github.com/w3c/webrtc-encoded-transform/issues/220